src/app/docs/summary-action-bar/summary-action-bar-docs.component.html

```diff
@@ -0,0 +1,135 @@
+          Don't use summary action bars to wrap a large amount of content will eat into the vertical space on the page. Show the least amount of information that the user needs to inform their actions. Explore alternate layouts if you need to show a large amount of summary information.
+        </p>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Anatomy"
+    >
+      <sky-docs-anatomy
+        imagePath="~/assets/img/anatomy.png"
+      >
+        <sky-docs-anatomy-item>Actions</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item isOptional="true">Summary information</sky-docs-anatomy-item>
+      </sky-docs-anatomy>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Behavior and states"
+    >
+      <sky-docs-design-guideline
+        heading="Collapsing summary information"
+      >
+        <p class="sky-docs-text-block">
+          In smaller viewports the summary information section becomes expandable/collapsible to allow more visible space for the main content.
+        </p>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Content"
+    >
+      <sky-docs-design-guideline
+        heading="Summary information"
+      >
+        <p class="sky-docs-text-block">
+          In cases where the user needs to see summarized information about the selected items in a list or the data entered in a form as a confirmation before taking an action, put the relevant information in the summary information area.
+        </p>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Conditional actions"
+      >
+        <p class="sky-docs-text-block">
+          In multiselect lists, items may be in mixed states where certain actions can only be taken on certain items. Expose all the possible actions, and append the number of items that action will be taken on when invoked.
+        </p>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Secondary actions"
+      >
+        <p class="sky-docs-text-block">
+          Secondary actions will collapse into a context menu at small viewport widths.
+        </p>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Summary information"
+      >
+        <p class="sky-docs-text-block">
+          Summary information should reflow based on available width.
+        </p>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Related information"
+    >
+      <sky-docs-design-guideline
+        heading="Components"
+      >
+        <ul>
+          <li>
+            <a href="https://developer.blackbaud.com/skyux/components/modal">Modal</a>
+          </li>
+          <li>
+            <a href="https://developer.blackbaud.com/skyux/components/grid">Grid</a>
```

Comment:
We usually organize the components in this list in alphabetical order.

Reply:
Done